src/xlat-i/xi-Entry.C

```diff
@@ -1094,7 +1094,7 @@ void Entry::genTramInstantiation(XStr& str) {
           << "    }\n"
           << "    CProxy_" << container->tramInstances[i].type.c_str() << " tramProxy =\n"
           << "    CProxy_" << container->tramInstances[i].type.c_str()
-          << "::ckNew(2, dims, gId, itemsPerBuffer, false, 10.0);\n"
+          << "::ckNew(1, dims, gId, itemsPerBuffer, false, 0.01);\n"
```

Comment:
How was the period here selected? This is a significant change

Reply:
The 10.0 value was far too high, and was a remnant from some experiments. Ideally, this value should be set by the user(probably via Justin's attribute parameter change), not somewhere in the depths of charmxi. The same goes for the buffer size and associated fractions.